Iceberg-UI.package/IceRepositoriesBrowser.class/instance/addPackageToRepository..st

```diff
@@ -9,4 +9,6 @@ addPackageToRepository: repo
 	options := (MCWorkingCopy allManagers difference: repo workingCopies).
 
 	(self class chooseFrom: options displaying: descriptionBlock) 
-		ifNotNil: [ :workingCopy | repo addPackageNamed: workingCopy packageName ]
\ No newline at end of file
+		ifNotNil: [ :workingCopy | 
+			workingCopy modified: true.
```

Comment:
I would prefer this logic extracted to a method other than start putting this inside the block.